Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -423,6 +425,9 @@ def modulesOSForbidden (opts : LinterOptions) (modules : Array Lean.Name) : IO N
         else if s.contains '.' then
           isBad := true
           IO.eprintln s!"error: module name '{name}' contains forbidden character '.'"
+        else if s.contains ' ' || s.contains '\t' then
```

Comment:
Not sure, but should we check for `'\n'` and similar as well?

bors d+

Reply:
Good idea; added!